ci/Dockerfile.assets.server

```diff
@@ -88,7 +88,7 @@ RUN mkdir -p /tmp/final/global && cp /juno/README.md /tmp/final/global/
 
 WORKDIR /tmp/final
 
-# ADD generate_importmap.mjs /juno/ci/scripts/generate_importmap.mjs
+ADD generate_importmap.mjs /juno/ci/scripts/generate_importmap.mjs
```

Comment:
I think this wouldn't work cause the build context isn't the juno repo

Reply:
I copy the script over here [`4a182d7` (#632)](https://github.com/sapcc/juno/pull/632/commits/4a182d7ed02dd328f3b3d13a633316dc4c94148b#diff-02038500349a9ed27723ad33eaeb70ebe40b98addfcc94d3fc78bf7ceb37aa1cR483) and I tested it in the pipeline already.